src/vrs.Util.js

```diff
@@ -27,36 +27,74 @@ Ext.format = Ext.util.Format.format;
 // --------- Mouse and Keyboard support ------ //
 
 /**
-* Add hooks to intercept mouse wheel events.
-*
-* see: http://goo.gl/q9LIf
-*/
-vrs.addMouseWheelHooks = function() {
-   // Native scrolling in Browser
-   document.addEventListener('mousewheel', function(e){
-      var el = e.target;
-      var offset, scroller, _results;
-      _results = [];
+ * Add hooks to intercept mouse wheel events.
+ *
+ * see for base concept:
+ * https://www.sencha.com/forum/showthread.php?110792-Add-support-for-mouse-scrolling-%28for-development%29&p=744700&viewfull=1#post744700
+ */
+(function() {
+   // `wheel` events can have different modes to indicate how what the delta means.
+   // For our use here we simply need to know how to convert the delta into a
+   // reasonable pixel size for the scroller.
+   var delta_multipliers = {
+      0: 0.5,    // pixels
+      1: 12,     // lines
+      2: 12 * 20 // pages
+   };
+
+   function adjustScroller(scroller, evt) {
+      var multiplier = delta_multipliers[evt.deltaMode] || 0.5,
+          offset = {
+             x: 0,
+             y: evt.deltaY * multiplier
+          };
+
+		scroller.fireEvent('scrollstart', scroller, scroller.position.x, scroller.position.y, evt);
+		scroller.scrollBy(offset.x, offset.y);
+      console.log('asdf');
+		scroller.snapToBoundary();
+		scroller.fireEvent('scrollend', scroller, scroller.position.x, scroller.position.y - offset.y);
+   }
+
+   function findScroller(el) {
+      var cmp, scroller;;
+
       while (el !== document.body) {
-	 if (el && el.className && el.className.indexOf('x-container') >= 0) {
-	    var cmp = Ext.getCmp(el.id);
-	    if (cmp && typeof cmp.getScrollable == 'function' && cmp.getScrollable()){
-	       var scroller = cmp.getScrollable().getScroller();
-	       if (scroller) {
-		  var offset = {x:0, y: -e.wheelDelta*0.5};
-		  scroller.fireEvent('scrollstart', scroller, scroller.position.x, scroller.position.y, e);
-		  scroller.scrollBy(offset.x, offset.y);
-		  scroller.snapToBoundary();
-		  scroller.fireEvent('scrollend', scroller, scroller.position.x, scroller.position.y-offset.y);
-		  break;
-	       }
-	    }
-	 }
-	 _results.push(el = el.parentNode);
+         if (el && el.className && el.className.indexOf('x-container') >= 0) {
+            cmp = Ext.getCmp(el.id);
+	         if (cmp && typeof cmp.getScrollable == 'function' && cmp.getScrollable()) {
+	            scroller = cmp.getScrollable().getScroller();
+               if (scroller) {
+                  return scroller;
+               }
```

Comment:
Looks like this file always had tabs and spaces which makes sense it is a bunch of code copied out of various forum posts.